addons/uh60_weapons/functions/fnc_HARMGuidence.sqf

```diff
@@ -0,0 +1,92 @@
+#include "script_component.hpp"
+
+/*
+ * Author: Donov & Yax
+ * Flies the missile to a target, or scans for targets to hit
+ *
+ * Arguments:
+ * 0: Projectile <OBJECT>
+ * 1: Ammo <OBJECT>
+ * 
+ * Return Value:
+ * 0: Closest Target (or objNull for no Target) <OBJECT>
+ *
+ * Example:
+ * [_projectile, _ammo] call vtx_uh60_weapons_fnc_getHARMTarget
+ */
+
+params ["_projectile", "_ammo"];
+
+[{	//start of Yax's ITC APK Code
+		(_this select 0) params ["_projectile", "_ammo", "_foundTarget", "_launchTime","_lastFrameTime", "_targetPos"];
+		//Target pos needs to get passed as a parameter or this does not work
+		private _frameTime = time - _lastFrameTime;
+		(_this select 0) set [4, time];
+
+		if (!alive _projectile) exitWith {
+			[_this select 1] call CBA_fnc_removePerFrameHandler;
+		};
+
+		//Start of Donov HARM Guidence computer
+		//systemChat "Missile Alive";
+		_missileTgtPos = [getPosASL _projectile, vectorDir _projectile, _projectile, vehicle _gunner] call ace_laser_fnc_shootRay;
+		/*  Using Shoot ray to determine a spot in space where the missile is looking in front of it 
+		 *  Then using nearestObjects to pull a list of "AllVehicles" from the radius of the seekerhead
+		 *  Then sorting through the list of "AllVehicls" and setting the target as the first vehicle (closest) in the list
+		 *		Only choosing vehicles that have radar on (turning off radar and moving should save the vehicle (in theory))
+		*/
+		if (!_foundTarget) then {
+			if ((_missileTgtPos # 1) != 0) then {
+				_target = [_projectile, ((_missileTgtPos # 1)*5.6712818 + 10), (_missileTgtPos # 0)] call vtx_uh60_weapons_fnc_HARMGuidence; 
```

Comment:
this is the function calling itself, did you test this?

Reply:
The code on my machine at home had the fnc_getHarmTarget being called.  It looks like I never pushed that repo to git.  It has been resolved now